style.css

```
@@ -52,7 +62,14 @@ h1#logo small a{font-size:12px;border-bottom:1px solid #FFF;}
 * html #content-body{float:left;margin-right:24px;} *+html #content-body{float:left;margin-right:28px;}
 #footer{height:8px;}
 #footer .text{color:#8D837B;font-size:10px;display:block;padding:12px;}
-#footer .top{position:relative;right:5px;top:-5px;display:inline;float:right;}
+
+#footer .top{
+	position:relative;right:5px;top:-5px; 
+	/* display:inline;float:right; */
+	display: block; text-align: right; color: #c0c0c0;
+	/* desbest edit */
```

Irrelevant comments. Dead code should be removed, and we don't need line-by-line authorship credits.

```suggestion
	display: block; text-align: right; color: #c0c0c0;
```